include/pfs/drp/stella/Spectra.h

```diff
@@ -3,28 +3,13 @@
 
 #include <vector>
 #include <iostream>
-#include <cmath>
-//#pragma clang diagnostic push
-//#pragma clang diagnostic ignored "-Wunused-variable"
-//#pragma clang diagnostic pop
-#include "boost/algorithm/string/trim.hpp"
-#include "lsst/base.h"
-#include "lsst/afw/geom.h"
-#include "lsst/afw/image/MaskedImage.h"
+#include "Controls.h"
 #include "lsst/log/Log.h"
-#include "lsst/pex/config.h"
 #include "lsst/pex/exceptions.h"
```

Comment:
`pexExcept` IS being used in `SpectraTemplates.hpp`. The reason why removing it doesn't break the code is that it is also being included in `CurveFitting.h`